App Force close when click Power Source tool

**Actual Behaviour**

App force close

**Expected Behaviour**

Generates programmable voltage and currents

**Steps to reproduce it**

Click on Power Souce tool

**LogCat for the issue**

> 2018-12-09 16:19:01.532 16722-16722/io.pslab E/AndroidRuntime: FATAL EXCEPTION: main
    Process: io.pslab, PID: 16722
    java.lang.RuntimeException: Unable to start activity ComponentInfo{io.pslab/io.pslab.activity.PowerSourceActivity}: java.lang.NumberFormatException: For input string: "-5,00"
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2778)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2856)
        at android.app.ActivityThread.-wrap11(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1589)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6494)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:440)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
     Caused by: java.lang.NumberFormatException: For input string: "-5,00"
        at sun.misc.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2043)
        at sun.misc.FloatingDecimal.parseFloat(FloatingDecimal.java:122)
        at java.lang.Float.parseFloat(Float.java:452)
        at java.lang.Float.valueOf(Float.java:417)
        at io.pslab.activity.PowerSourceActivity.limitDigits(PowerSourceActivity.java:637)
        at io.pslab.activity.PowerSourceActivity.onCreate(PowerSourceActivity.java:161)
        at android.app.Activity.performCreate(Activity.java:7009)
        at android.app.Activity.performCreate(Activity.java:7000)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1214)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2731)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2856) 
        at android.app.ActivityThread.-wrap11(Unknown Source:0) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1589) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:164) 
        at android.app.ActivityThread.main(ActivityThread.java:6494) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:440) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807) 

**Screenshots of the issue**

![screenshot_peluncur_pixel_20181209-162415](https://user-images.githubusercontent.com/32903387/49695601-1f32ea80-fbd0-11e8-98e6-7d5e30f8b14a.png)

**Would you like to work on the issue?**

Sorry, no

--- a/app/src/main/java/io/pslab/activity/PowerSourceActivity.java
+++ b/app/src/main/java/io/pslab/activity/PowerSourceActivity.java
@@ -640,7 +640,11 @@ public class PowerSourceActivity extends AppCompatActivity {
      * @return truncated float value
      */
     private float limitDigits(float number) {
-        return Float.valueOf(String.format(Locale.getDefault(), "%.2f", number));
+        try {
+            return Float.valueOf(String.format(Locale.getDefault(), "%.2f", number));
+        } catch (NumberFormatException e) {
+            return 0.00f;
+        }
     }
 
     /**

